example/train.py

```diff
@@ -0,0 +1,32 @@
+from keras.datasets import cifar10
+from keras.models import model_from_json
+import sys
+
+# Get the command line arguments
+model_file_name = ''
+try:
+    model_file_name = sys.argv[1]
+except IndexError:
+    print('Usage: python train.py model_json_file')
+    exit()
+
+# Load the dataset (keras.datasets.cifar10)
+(x_train, y_train), (x_test, y_test) = cifar10.load_data()
+
+# Load the model from JSON file
+json_file = open(model_file_name, 'r')
+loaded_model_json = json_file.read()
+json_file.close()
+loaded_model = model_from_json(loaded_model_json)
+
```

Comment:
```suggestion
loaded_model.summary()
```
You can print the model summary here using the above code. Add a one line comment to explain what this does. Optionally, you can get input from the user to either enable or disable printing model summary.